### PR TITLE
Temporarily disable phone number

### PR DESCRIPTION
**Pull request by robinandeer**
## What?

Implement feature flag for customer service phone  …
Add color prop to correctly style language picker.
Update snapshots.

## Why?

Key solutions not available during x-mas.

**Ticket(s): []**

### CODE REVIEW

**Review by guilhermespopolin — APPROVED** [review]
🤔 So in order to show/remove the phone number from the App we'd need to configure a env variable and rebuild the app? Would be possible to get customer service availability from an api?

**Comment by robinandeer** [comment]
@guilhermespopolin maybe an API will be overkill but would be nice to have an interface where other people can toggle the default state of features like this. Much like they can add content in the CMS.

I talked to @styrelius about making that part of the project to setup A/B testing!

**Comment by robinandeer** [comment]
@styrelius here's the review app: https://web-onboardi-hotfix-tem-yusvbv.herokuapp.com/se/new-member/

**Comment by robinandeer** [comment]
Seems to work as expected

Screenshot 2021-12-23 at 10 11 49
Screenshot 2021-12-23 at 10 11 37
Screenshot 2021-12-23 at 10 10 49

File: src/client/pages/Embark/LanguagePicker.tsx [hunk 1]
```diff
@@ -5,51 +5,63 @@ import { Link, useLocation } from 'react-router-dom'
 import { useMarket } from 'components/utils/CurrentLocale'
 import { languagePickerData } from './languagePickerData'
 
+type ColorProps = {
```
Comment (dendoesit) [inline src/client/pages/Embark/LanguagePicker.tsx:8]: this is great. 👍

**Review by styrelius — APPROVED** [review]
👌

**Pull request merged**